### PR TITLE
levels strings

### PR DESCRIPTION
**Pull request by FnrDev**
**Please describe the changes this PR makes and why it should be merged:**

**In this PR:**

### CODE REVIEW

File: en/bot.json [hunk 1]
```diff
@@ -484,15 +484,16 @@
...
-	"setxp_remove_xp_description": "The amount of xp you want to remove from the user.",
+	"setxp_type_description": "Type of xp either text or voice",
+	"setxp_user_description": "The user you want to add xp to.",
+	"setxp_xp_description": "The amount of xp to add to the user.",
 	"setxp_changed_xp": "**✅ | XP has been updated for @{0}**",
-	"setxp_select_type": "**Please select the type of xp.**",
-	"setxp_text_select": "Adds xp to the text level",
-	"setxp_voice_select": "Adds xp to the voice level",
-	"setxp_premium": ":x: This command is only available for premium subscribers."
+	"setlevel_confirm_message": ":thinking: **Are you sure you want to add level for {0}**",
```
Comment (FnrDev) [inline en/bot.json:491]:
```suggestion
	"setlevel_confirm_message": ":thinking: **Are you sure you want to add level for @{0}**",
```

File: en/bot.json [hunk 2]
```diff
@@ -484,15 +484,16 @@
...
+	"setxp_premium": ":x: This command is only available for premium subscribers.",
+	"setlevel_description": "Changes member's level",
+	"setlevel_user_description": "The user you want to add level to.",
+	"setlevel_type_description": "Type of level to add for",
+	"setlevel_level_description": "The level to add to the user.",
+	"setlevel_max_level": ":x: Max level to add is {0}",
+	"setxp_confirm_message": ":thinking: **Are you sure you want to add xp for {0}**"
```
Comment (FnrDev) [inline en/bot.json:498]:
```suggestion
	"setxp_confirm_message": ":thinking: **Are you sure you want to add xp for @{0}**"
```

File: en/bot.json [hunk 3]
```diff
@@ -484,15 +484,16 @@
...
 	"button_someone_else": ":x: This button belongs to someone else.",
 	"setxp_description": "Changes member's xp",
-	"setxp_add_description": "Add xp to user.",
-	"setxp_add_user_description": "The user you want to add xp to.",
-	"setxp_add_xp_description": "The amount of xp to add to the user.",
-	"setxp_remove_description": "Remove xp from a user",
-	"setxp_remove_user_description": "The user you want to remove xp from",
-	"setxp_remove_xp_description": "The amount of xp you want to remove from the user.",
+	"setxp_type_description": "Type of xp either text or voice",
+	"setxp_user_description": "The user you want to add xp to.",
```
Comment (Anasqx) [inline en/bot.json:488]:
```suggestion
	"setxp_user_description": "The user to whom you want to add xp",
```

File: en/bot.json [hunk 4]
```diff
@@ -484,15 +484,16 @@
...
+	"setxp_xp_description": "The amount of xp to add to the user.",
 	"setxp_changed_xp": "**✅ | XP has been updated for @{0}**",
-	"setxp_select_type": "**Please select the type of xp.**",
-	"setxp_text_select": "Adds xp to the text level",
-	"setxp_voice_select": "Adds xp to the voice level",
-	"setxp_premium": ":x: This command is only available for premium subscribers."
+	"setlevel_confirm_message": ":thinking: **Are you sure you want to add level for @{0}**",
```
Comment (Anasqx) [inline en/bot.json:491]:
```suggestion
	"setlevel_confirm_message": ":thinking: **Are you sure that you want to add a level for @{0}**?",
```

File: en/bot.json [hunk 5]
```diff
@@ -484,15 +484,16 @@
...
+	"setxp_premium": ":x: This command is only available for premium subscribers.",
+	"setlevel_description": "Changes member's level",
```
Comment (Anasqx) [inline en/bot.json:493]:
```suggestion
	"setlevel_description": "Changes member level",
```

File: en/bot.json [hunk 6]
```diff
@@ -484,15 +484,16 @@
...
+	"setlevel_user_description": "The user you want to add level to.",
```
Comment (Anasqx) [inline en/bot.json:494]:
```suggestion
	"setlevel_user_description": "The user to whom you want to add a level",
```

File: en/bot.json [hunk 7]
```diff
@@ -484,15 +484,16 @@
...
+	"setlevel_type_description": "Type of level to add for",
```
Comment (Anasqx) [inline en/bot.json:495]:
```suggestion
	"setlevel_type_description": "Level type to add",
```

File: en/bot.json [hunk 8]
```diff
@@ -484,15 +484,16 @@
...
+	"setlevel_level_description": "The level to add to the user.",
```
Comment (Anasqx) [inline en/bot.json:496]:
```suggestion
	"setlevel_level_description": "The level to be added to the user",
```

File: en/bot.json [hunk 9]
```diff
@@ -484,15 +484,16 @@
...
+	"setlevel_max_level": ":x: Max level to add is {0}",
```
Comment (Anasqx) [inline en/bot.json:497]:
```suggestion
	"setlevel_max_level": ":x: The maximum level that can be added is {0}",
```

File: en/bot.json [hunk 10]
```diff
@@ -484,15 +484,16 @@
...
-	"setxp_add_description": "Add xp to user.",
-	"setxp_add_user_description": "The user you want to add xp to.",
-	"setxp_add_xp_description": "The amount of xp to add to the user.",
-	"setxp_remove_description": "Remove xp from a user",
-	"setxp_remove_user_description": "The user you want to remove xp from",
-	"setxp_remove_xp_description": "The amount of xp you want to remove from the user.",
+	"setxp_type_description": "Type of xp either text or voice",
+	"setxp_user_description": "The user you want to add xp to.",
+	"setxp_xp_description": "The amount of xp to add to the user.",
 	"setxp_changed_xp": "**✅ | XP has been updated for @{0}**",
```
Comment (Anasqx) [inline en/bot.json:490]:
```suggestion
	"setxp_changed_xp": "**✅ @{0}**' XP has been updated",
```

**Review by Anasqx — CHANGES_REQUESTED** [review]
Improvements

**Pull request merged**